### ISSUE TITLE
Add Coconut line numbers to MyPy output

### ISSUE THREAD

**Issue by evhub** [evhub/coconut#332]
Should automatically add Coconut line numbers to Python line numbers in `--mypy` output.

**Comment by arnauorriols** [comment]
Any specific approach you have in mind? 

I'm affected by this issue myself, and I wanted to give the PR a go. After some consideration, I have the feeling the only correct way to implement this is to typecheck the coconut files, instead of the python output...

**Comment by evhub** [comment]
@arnauorriols The easy way to do this is to only have it enabled when `--line-numbers` is passed (which should also then be turned on by default when `--mypy` is passed). Then, simply look for the line number comments in the resulting MyPy error messages and use those to figure out the original Coconut line to replace them with.

**Comment by zain93393** [comment]
I would like to work on this issue. Can you please guide me about the issue.

**Comment by CristhianMotoche** [comment]
Hello there! Is this still an open issue?

**Comment by evhub** [comment]
Yes, this is still open. I don't have time to guide anyone through this in depth, but feel free to submit a PR if you want.
